cms/jinja2/templates/base.html

```diff
@@ -8,72 +8,80 @@
 {% endset %}
 
 {# fmt:off #}
-{% set languages = languages | default([
-    {
-        "url": "#",
-        "isoCode": "en",
-        "text": "English",
-        "current": true
-    },
-    {
-        "url": "#",
-        "isoCode": "cy",
-        "text": "Cymraeg",
-        "current": false
-    }
-])
-%}
+{% set languages=get_translation_urls() %}
+{% set hreflangs=get_hreflangs() %}
 
 {% if page %}
     {% set body_class = ["template-", page.get_verbose_name()| lower | replace(" ", "-")] | join %}
 {% else %}
     {% set body_class = '' %}
 {% endif %}
 
+{#- Untranslated pages will have aliases to the original page -#}
+{%- if page.alias_of and request.LANGUAGE_CODE != page.alias_of.locale.language_code -%}
+    {%- set localised_version_not_available = true -%}
+{%- endif -%}
+
```

Comment:
nitpick: move to `base_page.html`

Reply:
Addressed in [642e6a3](https://github.com/ONSdigital/dis-wagtail/pull/140/commits/642e6a39e1a1bbe99350c7bc03782f29eb0fd165)